### PR TITLE
Tco data

### LINKED ISSUE #23: Support including TCO data in schemaless generation

### PR DESCRIPTION
**Pull request by sbuss**
Fixes #23

### CODE REVIEW

File: schemaless/create_schemaless.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    parser.add_argument('--pts_file', help='PTS file', default='')
+    parser.add_argument('--tco_file', help='TCO file', default='')
+    parser.add_argument('--out_file', help='output file for schemaless csv')
+
+    parser.add_argument(
+        '--diff',
+        help='A schemaless csv generated by this script, to diff against.',
+        default='')
+    parser.add_argument(
+        '--the-date',
```
Comment (ilakyapal) [inline schemaless/create_schemaless.py]: Just curious when this would be useful?

Comment (sbuss) [reply schemaless/create_schemaless.py]: It's for stable testdata generation. This is actually part of #30

File: schemaless/create_uuid_map.py [hunk 2]
```diff
@@ -152,12 +157,12 @@ def values(self):
...
     def __init__(self,
                  record_id,
-                 date_opened=None,
+                 date=None,
                  parents=None,
                  children=None,
                  uuid=None):
         self.record_id = record_id
-        self.date_opened = date_opened
+        self.date = date
```
Comment (ilakyapal) [inline schemaless/create_uuid_map.py]: I'm not sure what "date" is expected to be for a Node? "date_opened" feels clearer?

File: schemaless/create_uuid_map.py [hunk 3]
```diff
@@ -2,11 +2,11 @@
...
-from datetime import datetime
 import uuid
 
 from fileutils import open_file
-from create_schemaless import latest_values
+from schemaless.create_schemaless import latest_values
+from schemaless.sources import source_map
 
 
 class RecordGraph:
```
Comment (ilakyapal) [inline schemaless/create_uuid_map.py]: So just to clarify, are you envisioning the RecordGraph would have nodes corresponding to records from all sources (including PTS, TCO in addition to PPTS)? And it's not necessarily true that the root record for a given project would always be a node corresponding to a PPTS PRJ (like for some project that goes from OCII --> DBI)?

File: schemaless/create_uuid_map.py (hunk 2 above)
Comment (sbuss) [reply schemaless/create_uuid_map.py]: different sources have different dates, eg TCO is date_issued. `date` here is just for choosing the most recent value for a given Node's parents/children

File: schemaless/create_uuid_map.py (hunk 3 above)
Comment (sbuss) [reply schemaless/create_uuid_map.py]: Right, Nodes are not necessarily PRJs or chlild records. This graph can/should be used to build the entire tree for UUID calculation purposes

Comment (ilakyapal) [reply schemaless/create_uuid_map.py]: ok great, makes sense to me

File: schemaless/create_schemaless.py (hunk 1 above)
Comment (ilakyapal) [reply schemaless/create_schemaless.py]: ok got it, not for real life pipeline stuff

**Pull request merged**